lighthouse-core/test/fixtures/fraggle-rock/reports/sample-flow-result.json

```diff
@@ -8537,12 +8543,12 @@
               ],
               "items": [
                 {
-                  "url": "https://www.mikescerealshack.co/_next/static/chunks/framework.9d524150d48315f49e80.js",
+                  "url": "https://www.mikescerealshack.co/",
```

Comment:
what audit is this second chunk?

Reply:
`long-tasks`. Similar to `bootup-time`, it was getting an empty `frames` array here: https://github.com/GoogleChrome/lighthouse/blob/90775ace823d183ed2c6ef13ca1f28bc139dccf9/lighthouse-core/computed/main-thread-tasks.js#L19